web/styles/informational_overlays.css

```diff
@@ -1,108 +1,186 @@
 .informational-overlays {
+    display: grid;
+    place-items: center;
+
     .overlay-content {
-        /* because zoom breaks at 525px perhaps due to rounding errors, so add a
-           trivial amount of width so it doesn't break. */
-        width: 550px;
+        width: 90%;
+        max-width: 650px;
         margin: 0 auto;
         position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
+        top: unset;
+        border-radius: 6px;
         overflow: hidden;
 
         background-color: var(--color-background-modal);
     }
 
     .overlay-tabs {
-        padding: 10px 0;
-        border-bottom: 1px solid hsl(0deg 0% 0% / 20%);
+        position: relative;
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        gap: 15px;
+        padding: 0 5%;
+        height: 60px;
+        box-shadow: 0 3px 12px 1px hsl(207deg 100% 11% / 8%);
+        z-index: 2;
 
         .tab-switcher {
-            margin-left: 15px;
+            flex-grow: 1;
         }
 
         .exit {
-            float: right;
-            font-size: 1.5rem;
-            color: hsl(0deg 0% 67%);
-            font-weight: 600;
-            margin: 0 15px;
+            display: inline-block;
```

Comment:
So in this case, `.exit` is a flex item of `.overlay-tabs`. And because it's a flex item by itself, there's no need to tell it to display inline. If we just let flexbox do its thing, you can actually ditch almost all of these properties! I'll let you try this yourself to see the result, but there's no change:

<img width="300" alt="Screenshot 2023-06-16 at 11 16 56 AM" src="https://github.com/zulip/zulip/assets/170719/b86af47c-e6ba-4c9f-a86e-7b80225866ac">